core/src/test/java/io/questdb/test/griffin/CreateTableDedupTest.java

```diff
@@ -262,7 +286,11 @@ public void testEnableDedupDroppedColumnColumnConcurrently() throws Exception {
                             .ofDropColumn("sym");
 
                     tw1.apply(dropColumnAlterBuilder.build(), true);
-                    tw2.apply(ao, true);
+                    try {
+                        tw2.apply(ao, true);
```

Comment:
Would be great to add `Assert.fail();` after this line as we expect an exception here.

Reply:
added